ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -194,11 +201,38 @@ export interface Data extends RaidbossData {
 const triggerSet: TriggerSet<Data> = {
   id: 'AnabaseiosTheTwelfthCircleSavage',
   zoneId: ZoneId.AnabaseiosTheTwelfthCircleSavage,
+  config: [
+    {
+      id: 'engravement1DropTower',
+      name: {
+        en: 'Paradeigma: Engravement of Souls Tower strategy',
+        cn: '第一次拉线踩塔方法',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Vertical Beam (Game8)': 'sharp',
+          'Tilt Beam': 'tilt',
+          'Only tower color': 'tower',
```

Comment:
Hmm, sorry for wording quibbling here, but I don't think these are descriptive enough, and I think "tilt" isn't precise either (you can have the tethers be vertical but then still go clockwise always vs quadrant).

I'm also a little hesitant to prescribe a particular strategy here too, and wonder if `tower` should be the default.  However, I do think that the "nearest quadrant" tower appears to be the aether PF strat as well, so I think either `tower` or `quadrant` could be ok as the default.

Hmm, also this points out maybe a ui issue where if we start having more strategy options I almost need some place to put descriptions of what these are or links to raidplans? For now, I think it's better to be extra wordy.  (I don't know what Game8 is.  If you have a link to a raidplan or something, can you leave it in a comment?)

```suggestion
          'Vertical beam + nearest quadrant tower (Game8)': 'quadrant',
          'Clockwise tower from beam': 'clockwise',
          'No strategy: just call tower color': 'tower',
```

If we end up with more of these strategy decisions, maybe in the future we need a notification or the cactbot config ui could prompt people that they have options they need to set as well.  This is somewhat of a new feature.

Reply:
There's also Inumaru "add-relative" strat too, but that could be a follow-up if anybody else wanted to add it: https://www.reddit.com/r/ffxiv/comments/1469jxz/p12s_para_2_first_tether_set_inumaru_strat/

Reply:
Will, in Japan and China we often use game8's raidplan, links follow 
https://game8.jp/ff14/534748

And for the 'clockwise' raidplan, I saw it in a Youtube video:
https://www.youtube.com/watch?v=Q4Qy3CRMdQc&ab_channel=Rinon

Considering that there are many kinds of raidplans, it is currently possible to set the default to 'tower'.